serializers/uri-builder.js

```diff
@@ -0,0 +1,37 @@
+const appRoot = require('app-root-path');
+const config = require('config');
+const url = require('url');
+
+const { protocol, hostname } = config.get('server');
+const api = appRoot.require('/package.json').name;
+
+/**
+ * @summary Self link builder
+ * @function
+ * @param {string} id
+ * @returns A self link URL
+ */
+const selfLink = id => url.format({
+  protocol,
+  hostname,
+  pathname: `/${api}/${id}`,
+});
```

Comment:
I think the port should be included in this self link

Reply:
Like we discussed in person, let's talk about this as a group as well.

Reply:
Any updates on this? @ianbrown9475 the self link is mainly for the clients to use, and they'll be making api calls directly to apigee so they'll be using port 443, so there's no need to show that.

Reply:
Make sense. I agreed that port shouldn't be exposed in any link object.